### PR TITLE
DateTimePicker - trigger onChange method when value is empty

### PR DESCRIPTION
**Pull request by lucechal14**
Scope: Update onChange trigger validation to allow trigger when the value of the DateTimePicker is empty

### CODE REVIEW

File: frontend/packages/core/src/Input/date-time.tsx [hunk 1]
```diff
@@ -26,8 +26,9 @@ const DateTimePicker = ({ onChange, ...props }: DateTimePickerProps) => (
     <MuiDateTimePicker
       renderInput={inputProps => <PaddedTextField {...inputProps} />}
       onChange={(value: Dayjs | null) => {
-        if (value && value.isValid()) {
-          onChange(value.toDate());
+        if (!value || value.isValid()) {
```
Comment (jdslaugh) [inline frontend/packages/core/src/Input/date-time.tsx:29]: Should we do this in the time picker as well?

**Review by jdslaugh — COMMENTED** [review]
Lets do a test deploy of this and make sure it doesn't have any issues with our current implementations of this.

File: frontend/packages/core/src/Input/date-time.tsx (hunk 1 above)
Comment (lucechal14) [reply frontend/packages/core/src/Input/date-time.tsx:29]: i don't have a scenario for the time picker where we need it but I can certainly do it too

**Review by jecr — APPROVED** [review]

**Review by jdslaugh — APPROVED** [review]
Looks good. We can address the TimePicker in a future PR if we wind up needing it.

**Pull request merged**